api/lib/models/index.ts

```diff
@@ -3,6 +3,8 @@ export interface CCDEventResponse {
     caseDetails: any
 }
 
+export * from './documents'
```

Comment:
No need to treat model like an index file  ie 
exporting an export from an included file is redundant 
import { DMDocuments} from  'api/lib/models/documents'  implicitly tells us where it's from and structure is imposed this way, there's no need for the artificial constraint